etl/cris_import/cr3_ocr_narrative.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+from datetime import datetime, timezone
+from concurrent.futures import ProcessPoolExecutor
+
+from pdf2image import convert_from_bytes
+from pytesseract import image_to_string, image_to_data
+
+from utils.cli import get_cli_args
+from utils.files import download_cr3_pdf
+from utils.graphql import (
+    make_hasura_request,
+    NARRATIVES_TODO_QUERY,
+    UPDATE_CRASH_NARRATIVE_OCR_MUTATION,
+)
+from utils.logging import init_logger
+from utils.process_pdfs import get_cr3_version
+from utils.settings import NARRATIVE_BBOX_PIXELS
+
+
+def extract_narrative_pdf(cris_crash_id, crash_pk, index):
+    """Handles narrative extraction of one PDF
+
+    Args:
+        cris_crash_id (int): the CRIS crash ID
+        crash_pk (int): the crash ID in the VZ database
+        index (int): a unique id which captures the position of this
+            item in the list of narratives being processed by the 
+            script. it enables better error logging during concurrency
+    """
+    logger.info(f"Processing cris crash ID {cris_crash_id} ({index})")
+    pdf = download_cr3_pdf(cris_crash_id)
+
+    logger.debug("Converting PDF to image...")
+    page = convert_from_bytes(
+        pdf,
+        fmt="jpeg",  # jpeg is much faster than the default ppm fmt
+        first_page=2,  # page 2 has the crash narrative
+        last_page=2,
+        dpi=150,
+    )[0]
+
+    cr3_version = get_cr3_version(page)
+    logger.debug(f"CR3 version: {cr3_version}")
+
+    # uncomment to save a copy of the pdf page image
+    # page.save(f"temp/{cris_crash_id}_page_{cr3_version}.jpeg")
+
+    bbox = NARRATIVE_BBOX_PIXELS[cr3_version]
+
+    logger.debug("Cropping narrative from PDF...")
+    narrative_image = page.crop(bbox)
+
+    # uncomment to save a copy of the cropped narrative image
+    # narrative_image.save(f"temp/{cris_crash_id}_narrative_{cr3_version}.jpeg")
+
+    logger.debug("Extracting narrative text...")
+    narrative = None
+    narrative = image_to_string(narrative_image, lang="eng")
```

Comment:
the old version of the script used a `try`/`except`/ block for this call, but it never errored for me, even when parsing some gnarly old garbage PDFs. my thinking was that for now it would be better to know if this breaks somehow, but if it becomes a nuisance we can easily revisit.